### DISCUSSION TITLE
npm package resolution fails when using react-native-tvos

### DISCUSSION THREAD

**Discussion by adamkoch** [react-native-tvos/react-native-tvos#761] [Q&A]
Hi all,

When using `react-native-tvos` for `react-native` in `package.json` like so:
```
"react-native": "npm:react-native-tvos@~0.74.3-0",
```

I now get a number of `ERESOLVE could not resolve` errors when running `npm install`, for example:
```
npm ERR! While resolving: @react-native-async-storage/async-storage@1.23.1
npm ERR! Found: react-native@0.74.3-0
npm ERR! node_modules/react-native
npm ERR!   react-native@"npm:react-native-tvos@~0.74.3-0" from the root project
npm ERR!   peer react-native@"*" from @react-native-firebase/app@20.3.0
npm ERR!   node_modules/@react-native-firebase/app
npm ERR!     @react-native-firebase/app@"^20.3.0" from the root project
npm ERR!     peer @react-native-firebase/app@"20.3.0" from @react-native-firebase/app-check@20.3.0
npm ERR!     node_modules/@react-native-firebase/app-check
npm ERR!       @react-native-firebase/app-check@"^20.3.0" from the root project
npm ERR!     4 more (@react-native-firebase/auth, ...)
npm ERR!   10 more (@react-native-tvos/virtualized-lists, ...)
```

To overcome this, I am using `npm install --force` which gives warnings, but installs fine and everything works.

Is there a way to tell `@react-native-async-storage/async-storage` and other modules that explicitly depend on a version of `react-native` that the alternative of `react-native-tvos` is an acceptable alternative? Or that it should just consider it `react-native`?

When I look inside `package.json` of say async-storage, I see this:
```
  "peerDependencies": {
    "react-native": "^0.0.0-0 || >=0.60 <1.0"
  },
```

But theoretically, that condition should be satisfied, given my `package.json` still depends on `react-native` and has a correct version within that range (even though it's a slightly different fork).

Thanks!

**Comment by douglowder** [comment]
Does this issue occur when using `yarn`?
